File: platforms/ios/Dezel/Dezel/Extensions/Date.swift
/**
 * @extension Date
 * @since 0.5.0
 * @hidden
 */
internal extension Date {

	/**
	 * @property iso
	 * @since 0.5.0
	 * @hidden
	 */
	var iso: String {
		return DateFormater.iso(self)
	}

	/**
	 * @method format
	 * @since 0.5.0
	 * @hidden
	 */
	func format(_ format: String, _ locale: String) -> String {
		return DateFormater.format(self, format: format, locale: locale)
	}
}

File: platforms/android/Dezel/dezel/src/main/java/ca/logaritm/dezel/extension/Date.kt
package ca.logaritm.dezel.extension

import ca.logaritm.dezel.date.DateFormater
import java.util.*

/**
 * @property Date.iso
 * @since 0.5.0
 * @hidden
 */
internal val Date.iso: String
	get() = DateFormater.iso(this)

internal fun Date.format(format: String, locale: String): String {
	return DateFormater.format(this, format, locale)
}